@xen-orchestra/web-core/lib/components/state-hero/VtsStateHero.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :class="type" class="vts-state-hero">
+  <div :class="[type, image]" class="vts-state-hero">
```

Comment:
From what I understand, the class `image` is added to change the background color when we display the "error" image.
But I see that the `VtsErrorNoDataHero` component also applies the same background color.
I think we have to choose which component should apply the styles in case of an error state.
Since `VtsErrorNoDataHero` is just a wrapper, maybe we can handle this error case directly here in `VtsStateHero`.

To improve the error state handling, I think we can make a _computed_ that would also handle the `busy` prop (IMO having the red background when `busy` is `true` does not make sense):
```ts
const error = computed(() => !props.busy && props.image === 'error-no-data')
```
Then in the template:
```html
<div :class="[type, { error }]" class="vts-state-hero">
```
And in the CSS:
```css
  &.error {
    background-color: var(--color-danger-background-selected);

    .loader,
    .text {
      color: var(--color-danger-txt-base);
    }
  }
```